libbpf-tools/runqslower.c

```diff
@@ -20,6 +21,8 @@ static volatile sig_atomic_t exiting = 0;
 struct env {
 	pid_t pid;
 	pid_t tid;
+	int task_comm_len;
```

Comment:
env can be seen as a kind of global variable, and I think it’s best to minimize its use unless absolutely necessary. In my opinion, task_comm_len doesn’t seem essential globally. 
If it’s still needed, at the very least, maintaining name consistency with comm seems necessary.